aten/src/ATen/cuda/detail/IndexUtils.cu

```diff
@@ -32,7 +32,7 @@ bool overlappingIndices(const Tensor& t) {
   // `dim`, or the innermost stride is 0.
 
   /* Extract size/stride arrays; only consider size >1 dims. */
-  SizeAndStride info[t.dim()];
+  SizeAndStride *info = (SizeAndStride *)malloc(sizeof(SizeAndStride) * t.dim());
```

Comment:
MSVC doesn't support variable length arrays, but not sure if this is the best way to allocate an array at runtime.

Reply:
Either use `alloca` or a `unique_ptr`